runners/direct-java/build.gradle

```diff
@@ -67,7 +66,6 @@ dependencies {
   dependOnProjects.each {
     implementation project(it)
   }
-  shadow project(path: ":sdks:java:core", configuration: "shadow")
```

Comment:
Ditto. Why did `direct-java` needed to depend on "core" before ?

Reply:
Now it depends on core above, in the `dependOnProjects` section.